samples/dnn/text_detection.cpp

```diff
@@ -138,7 +144,19 @@ int main(int argc, char** argv)
 
                 cvtColor(cropped, cropped, cv::COLOR_BGR2GRAY);
 
-                Mat blobCrop = blobFromImage(cropped, 1.0/127.5, Size(), Scalar::all(127.5));
+                // Preprocessing for text recognition
+                Mat blobCrop;
+                if(ocrFlag)  //Normalization in https://github.com/clovaai/deep-text-recognition-benchmark
+                {
+                    blobCrop = blobFromImage(cropped, 1.0/255.0, Size(), Scalar());
+                    blobCrop -= 0.5;
+                    blobCrop /= 0.5;
+                }
+                else  //Normalization in https://github.com/meijieru/crnn.pytorch
+                {
+                    blobCrop = blobFromImage(cropped, 1.0/127.5, Size(), Scalar::all(127.5));
+                }
```

Comment:
first case:
[0, 255] -> [0, 1] -> [-0.5, 0.5] -> [0, 1]
second:
[0, 255] -> [-127.5, 127.5] -> [0, 1]

They are equal

Reply:
@dkurt Thank you. 
I just tested it, these two preprocessing is indeed the same, I will remove the ```ocrFlag``` in the code.